contracts/consensus/Consensus.sol

```diff
@@ -25,8 +25,9 @@ import "../core/CoreI.sol";
 import "../core/CoreStatusEnum.sol";
 import "../reputation/ReputationI.sol";
 import "../proxies/MasterCopyNonUpgradable.sol";
+import "./CoreLifetime.sol";
```

Comment:
please sort imports, move this to L20

1. external imports
space
2. local imports
3. other modules imports

sorted alphabethically